ethcore/src/state.rs

```diff
@@ -56,6 +56,22 @@ impl AccountEntry {
 			AccountEntry::Missing => false,
 		}
 	}
+
+	fn clone_dirty(&self) -> Option<AccountEntry> {
```

Comment:
maybe add some docs for this function - difficult to immediately see what it's trying to do